analyzers/tests/SonarAnalyzer.UnitTest/TestCases/BlazorQueryParameterRoutableComponent_Noncompliant.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNetCore.Components;
+using System;
+
+public class Nullable
+{
+
+}
```

Comment:
Nipick to save some lines :smile:
```suggestion
public class Nullable { }
```